_methods/bodystorming.markdown

```diff
@@ -16,6 +17,9 @@ how: |
       <li>Review the exercise as a team. Document some of the potential opportunities/challenges that this exercise suggests.</li>
     </ol>
     <p>Tip: If you anticipate discomfort from participants, it may be helpful to assign roles beforehand and start with a basic script.</p>
+
+gov-context: 
+	<p> No PRA implications. Even when users are present, the PRA explicitly exempts direct observation and non-standardized conversation (e.g., not a survey) that bodystorming entails. (1320.3.H.3) That means: Storm away, no PRA needed! </p>
```

Comment:
If you're not using government employees, you'll need to observe standard precautions for archiving P.I.I.